src/service/xhr-impl.js

```diff
@@ -191,8 +193,8 @@ export class Xhr {
       init.headers['Content-Type'] = 'application/json;charset=utf-8';
```

Comment:
Somehow missed this before. Would you mind deleting this line, and updating the `'application/json'` above to be `'application/json;charset=utf-8'`?